pkgs/development/compilers/dtc/default.nix

```diff
@@ -109,6 +109,8 @@ stdenv.mkDerivation (finalAttrs: {
     # Checks are broken on aarch64 darwin
     # https://github.com/NixOS/nixpkgs/pull/118700#issuecomment-885892436
     !stdenv.isDarwin &&
+    # Checks are broken on x86_64 linux musl
+    !stdenv.hostPlatform.isMusl &&
```

Comment:
An explanation of the actual problem would be nice.

Also, if the problem is building a shared library, then this should work fine for non-static musl, right?  So the check should probably be `!stdenv.hostPlatform.isStatic`?

Reply:
> An explanation of the actual problem would be nice.

:+1: 

> Also, if the problem is building a shared library, then this should work fine for non-static musl, right? So the check should probably be `!stdenv.hostPlatform.isStatic`?

Yup, you're right, the tests pass on `pkgsMusl`. I wasn't aware that there's a difference between `pkgsStatic` and `pkgsMusl` on Linux, thanks for pointing that out!